benchmarks/CMakeLists.txt

```diff
@@ -96,3 +61,11 @@ ConfigureNVBench(STATIC_MULTIMAP_BENCH
   hash_table/static_multimap/retrieve_bench.cu
   hash_table/static_multimap/query_bench.cu
   hash_table/static_multimap/count_bench.cu)
+
+###################################################################################################
+# - dynamic_map benchmarks ------------------------------------------------------------------------
+ConfigureNVBench(DYNAMIC_MAP_BENCH
```

Comment:
https://github.com/sleeepyjack/cuCollections/blob/61d498cbca7b53a5403d1ed9bb0667f58e7949a5/benchmarks/CMakeLists.txt#L31-L35

This reminds me `ConfigureNVBench` could be renamed as `ConfigureBench`. And at line 35 in the link above. `./nvbenchmarks` could be just `./benchmarks`

Reply:
addressed in [53f2c66](https://github.com/NVIDIA/cuCollections/pull/276/commits/53f2c6684f7ad64442c12ec4cf3ef83c33c9bf29)